vortex-duckdb/src/exporter/dict.rs

```diff
@@ -27,12 +29,14 @@ struct DictExporter<I: NativePType> {
     value_id: usize,
 }
 
-pub(crate) fn new_exporter(
+pub(crate) fn new_exporter_with_flatten(
```

Comment:
I think I'd prefer to only keep on `new_exporter` func where we pass in the flatten bool explicitly at all times.